### DISCUSSION TITLE
New Glossary Terms Discussion

### DISCUSSION THREAD

**Discussion by BrainonSilicon** [aim-rsf/Glossary-of-Terms#3] [General]
Hello 👋 
---

This Discussion space has been set aside for folks to discuss terms that should be added to the Glossary of Terms, or to engage in discussions around the definitions of a term. 

Please read the [Contributing Guidelines](https://github.com/aim-rsf/Glossary-of-Terms/blob/main/docs/contributing.md) before contributing to our community. 

### Template for suggesting terms:
- suggested term
- definition
- citation (where the definition came from)

### Discussing Definitions 
- Remember that different groups may use terms differently to you, so be kind and considerate when engaging in discussion. 
- If a definition has a citation, please remember to add a link so that others may find it.

**Comment by BrainonSilicon** [comment]
Suggested term to add to the glossary

* PPIE 
* Public and Patient Involvement and Engagement 
* Patient and Public Involvement means that activities and research are carried out ‘with’ or ‘by’ members of the public or patients, rather than ‘to’, ‘about’ or ‘for’ them. Patient and Public Engagement​ is about ensuring that information and research outputs reach patients and the public.
* Citation: https://www.hdruk.ac.uk/about-us/patient-and-public-involvement-and-engagement/

**Reply by BobJasper** [reply]
Good idea

**Reply by Rabittes** [reply]
Great idea as we use this term alot

**Comment by BobJasper** [comment]
Lay summary
Tem we are avoiding using because it has religious/faith connotations, we find it preferable to use plain language summary.

**Reply by BrainonSilicon** [reply]
- **Term:** Plain English Summary
- **Definition:** A plain English summary is a clear, brief summary of the research that has been written for members of the public, rather than researchers or professionals. It should be written clearly and simply, without jargon and with an explanation of any technical terms {cite}`NIHR2022`. This is sometimes also known as a "Plain Language Summary" or "Lay Summary", however the term "Lay Summary" is no longer used by most communities. 

Citation
---
NIHR website https://www.nihr.ac.uk/documents/plain-english-summaries/27363

**Comment by Rabittes** [comment]
NIHR - as the funding body we explain this to people

**Comment by Rabittes** [comment]
Principal investigator - we discuss this with PPIE in terms of who is leading the project

**Comment by Rabittes** [comment]
Focus group - in terms of explaining different types of gathering research

**Comment by Rabittes** [comment]
Health inequality - Trying to explain why studies are being completed

**Reply by BrainonSilicon** [reply]
Is there a definition you had in mind? I found [this one](http://www.healthscotland.scot/health-inequalities/what-are-health-inequalities) from Public Health Scotland website

**Reply by Rabittes** [reply]
Yes I would use this to explain as a reference

**Comment by Rabittes** [comment]
Multiple long term health conditions

**Comment by BrainonSilicon** [comment]
* Citizen Science 
* Citizen Science is the practice of public participation and collaboration in scientific research. Citizens take an active role in the scientific process in collaboration with researchers; from forming research questions and methods, to curating data and disseminating findings. 
* Citation: https://www.nationalgeographic.org/encyclopedia/citizen-science/
